tests/unit/utils/test_component_tools.py

```diff
@@ -103,8 +110,9 @@ def test_get_meta_for_file_powershell(self):
             }
         }
         result = self.component_matcher.get_meta_for_file(u'lib/ansible/modules/windows/win_ping.ps1')
-        self.assertEqual(result[u'labels'], [u'windoez'])
-        self.assertEqual(result[u'maintainers'], [u'jborean93'])
+        assert result[u'labels'] == [u'windoez']
+        expected_maintainers = sorted([u'cchurch', u'jborean93'])
+        assert sorted(result[u'maintainers']) == expected_maintainers
 
```

Comment:
Is it worth adding a test of `ignore` in here as well?

Reply:
ignore: done
no authors: impossible
authors + botmeta: dunno how it's different from what we have now